ostree-simplified-installer.sh

```diff
@@ -0,0 +1,720 @@
+#!/bin/bash
+set -euox pipefail
+
+# work with mock osbuild-composer repo
+if [[ $# -eq 1 || $# -eq 2 ]]; then
```

Comment:
Same as above.

Reply:
Removed, thanks!
I just got another question, is there a nightly repo defined in the vm where the test case is running? I supposed so, but where did you set up this repo?